### PR TITLE
chore: fix a couple of typos and do some minor cleanup

### PR DESCRIPTION
**Pull request by j4hr3n**

### CODE REVIEW

**Comment by trygve-lie** [comment]
What is the rationale to replace `request` with `_`? I am struggling to see the value in that to be honest.

**Review by digitalsadhu — APPROVED** [review]

File: plugins/errors.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import httpError from "http-errors";
 import fp from "fastify-plugin";
 
 export default fp(async function errors(fastify) {
-  fastify.setErrorHandler((error, request, reply) => {
+  fastify.setErrorHandler((error, _, reply) => {
```
Comment (digitalsadhu) [inline plugins/errors.js:5]:
```suggestion
  fastify.setErrorHandler((error, request, reply) => {
```

File: plugins/lazy.js [hunk 2]
```diff
@@ -3,11 +3,11 @@ import fp from "fastify-plugin";
 export default fp(async function lazyScriptPlugin(fastify, { enabled, base }) {
   // inject live reload when in dev mode
   if (enabled) {
-    fastify.addHook("onSend", (request, reply, /** @type {string} */ payload, done) => {
+    fastify.addHook("onSend", (_, reply, /** @type {string} */ payload, done) => {
```
Comment (digitalsadhu) [inline plugins/lazy.js:6]:
```suggestion
    fastify.addHook("onSend", (request, reply, /** @type {string} */ payload, done) => {
```

File: plugins/podlet.js [hunk 3]
```diff
@@ -24,7 +24,7 @@ export default fp(async function podlet(
   const packageJson = JSON.parse(readFileSync(new URL("../package.json", import.meta.url), { encoding: "utf8" }));
   const podiumVersion = new SemVer(packageJson.dependencies["@podium/podlet"].replace("^", "").replace("~", ""));
 
-  fastify.get(podlet.manifest(), async (req, reply) => {
+  fastify.get(podlet.manifest(), async (_, reply) => {
```
Comment (digitalsadhu) [inline plugins/podlet.js:27]:
```suggestion
  fastify.get(podlet.manifest(), async (request, reply) => {
```

File: plugins/podlet.js [hunk 4]
```diff
@@ -54,7 +54,7 @@ export default fp(async function podlet(
 
   if (development) {
     // wrap the markup in a podlet render call to get templating when in dev mode
-    fastify.addHook("onSend", (request, reply, /** @type {string} */ payload, done) => {
+    fastify.addHook("onSend", (_, reply, /** @type {string} */ payload, done) => {
```
Comment (digitalsadhu) [inline plugins/podlet.js:57]:
```suggestion
    fastify.addHook("onSend", (request, reply, /** @type {string} */ payload, done) => {
```

File: plugins/scripts.js [hunk 5]
```diff
@@ -4,7 +4,7 @@ export default fp(async function scriptsPlugin(fastify, { enabled, base }) {
   // inject live reload when in dev mode
   if (enabled) {
     fastify.log.debug("custom client side scripting enabled");
-    fastify.addHook("onSend", (request, reply, /** @type {string} */ payload, done) => {
+    fastify.addHook("onSend", (_, reply, /** @type {string} */ payload, done) => {
```
Comment (digitalsadhu) [inline plugins/scripts.js:7]:
```suggestion
    fastify.addHook("onSend", (request, reply, /** @type {string} */ payload, done) => {
```

**Pull request merged**